src/tests/electron/main.test.ts

```diff
@@ -23,6 +24,11 @@ describe('Electron E2E', () => {
         expect(await app.browserWindow.isVisible()).toBe(true);
         expect(await app.client.getWindowCount()).toBe(1);
         expect(await app.webContents.getTitle()).toBe('Accessibility Insights for Mobile');
+
+        const cancelButton = '.window-footer-button-cancel';
```

Comment:
We do have a `selectors` file to save these selectors to maintain one common place for all the selectors. Not sure about electron, but for web; look at: [common-selectors.ts](https://github.com/microsoft/accessibility-insights-web/blob/master/src/tests/end-to-end/common/element-identifiers/common-selectors.ts).
I believe, we can follow the same pattern for electron.

Reply:
Added common-selectors.ts in iteration 2